### PR TITLE
RT-242: Tweaks for SupplyChain model fields

### PR DESCRIPTION
**Pull request by dev-digital**
**Changes**
##### SupplyChain
-  gets _archived_reason_ TextField
- _contact_name_ and _contact_email_ accept blanks

##### SupplyChain
- _impact_ accept black
- _start_date_ can be null and accepts blanks

##### Note: 
Migration needs to be applied

### CODE REVIEW

File: defend_data_capture/supply_chains/migrations/0014_auto_20210521_1146.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.2 on 2021-05-21 11:46
```
Comment (mforner13) [inline defend_data_capture/supply_chains/migrations/0014_auto_20210521_1146.py:1]: It'd be good to replace the the bit in the file name after `0014_` with a brief description of what the migration does. Makes it much easier to keep track of :)

File: defend_data_capture/supply_chains/test/test_models.py [hunk 2]
```diff
@@ -51,6 +53,14 @@ def test_no_archived_date_save_strategic_action():
     )
 
 
+def test_no_archived_date_save_supply_chain():
```
Comment (mforner13) [inline defend_data_capture/supply_chains/test/test_models.py:56]: Should the test be called `test_no_archived_reason_save_supply_chain`? :)

**Review by mforner13 — COMMENTED** [review]
Looks good, just two small tweaks suggested :)

File: defend_data_capture/supply_chains/migrations/0014_auto_20210521_1146.py (hunk 1 above)
Comment (dev-digital) [reply defend_data_capture/supply_chains/migrations/0014_auto_20210521_1146.py:1]: Yes, make sense.

File: defend_data_capture/supply_chains/test/test_models.py (hunk 2 above)
Comment (dev-digital) [reply defend_data_capture/supply_chains/test/test_models.py:56]: Yes :smile:

**Review by mforner13 — APPROVED** [review]
Looks good! Just a spelling mistake in the migration file in 'archived' :)

**Comment by dev-digital** [comment]
> Looks good! Just a spelling mistake in the migration file in 'archived' :)

Done.

**Review by NickFitz — APPROVED** [review]
👍

**Pull request merged**